CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- Added support for retrieving the `Punctuated` sequence of parameters in a `FunctionBody`
+- `FunctionBody::iter_parameters` is now deprecated in favour of `Punctated::iter` 
```

Comment:
Put this under ### Changed.

Reply:
Also, your comment isn't accurate--it's deprecated in favor of `parameters().iter`, not `Puncutated::iter` generally.